bundle/manifests/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -325,7 +335,7 @@ spec:
                 env:
                 - name: GIT_REPO_PATH
                   value: /git
-                image: quay.io/opdev/operator-certification-operator:1.0.2
+                image: quay.io/opdev/operator-certification-operator:latest
```

Comment:
Is using the `latest` tag intentional?

Reply:
Yes. A developer should not need to pass anything to make (other than maybe IMAGE_BUILDER=podman), in order for there to be no unexpected changes to the codebase, leaving confusion as to what should be committed.